bundle.js

```diff
@@ -25,6 +35,7 @@ __export(script_exports, {
   i_error_out: () => i_error_out
 });
 module.exports = __toCommonJS(script_exports);
+var import_react = __toESM(require("react"));
```

Comment:
where does this come from? are we providing react to all PDK code?

Reply:
my bad, this accidentally got added with some testing i was doing i think. I moved this PR back to draft as i'm re-working it along with getting react to work.

Reply:
also this is a test file i'm going to be deleting soon. there is some testing stuff on the top level

Reply:
Ok gotcha! just saw it was marked ready and was giving it a quick look -- but will keep eyes elsewhere while your drafting :smile: